### PR TITLE
DeleteShortcut Command, Improvised Personal Alias Command, Complete Tests Added 

### PR DESCRIPTION
**Pull request by shanmu9898**

### CODE REVIEW

**Comment by shanmu9898** [comment]
Closes #108 #100 #98 #95 #94

File: src/main/java/seedu/address/logic/commands/ShortcutCommand.java [hunk 1]
```diff
@@ -51,7 +58,41 @@ public CommandResult execute() throws CommandException {
         return new CommandResult(String.format(MESSAGE_SUCCESS));
     }
 
-    public List<ShortcutDoubles> getCommandsList() {
-        return commandsList;
+    /**
+     * Checks if the shortcut command is valid or not
+     * @return
```
Comment (randypx) [inline src/main/java/seedu/address/logic/commands/ShortcutCommand.java]: @return tag is empty? Same goes for the method below

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -184,6 +190,21 @@ public boolean removePerson(Person key) throws PersonNotFoundException {
         }
     }
 
+    /**
+     *
+     * @param commandShortcut
+     * @return
+     * @throws UniqueShortcutDoublesList.CommandShortcutNotFoundException
+     */
```
Comment (randypx) [inline src/main/java/seedu/address/model/AddressBook.java]: No explanation of method given. Empty @return tag again.

File: src/test/java/seedu/address/logic/commands/DeleteShortcutCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    private Model model;
+    private Model expectedModel;
+    private final String validtestingCommandWord = "list";
+    private final String validtestingShortcutWord = "l";
+    private final String invalidtestingCommandWord = "king";
+
```
Comment (randypx) [inline src/test/java/seedu/address/logic/commands/DeleteShortcutCommandTest.java]: Variable names need to be in camelCase, e.g. validTestingShortcutWord. also perhaps shorten it, e.g validCommandWord, validShortcut, invalidCommandWord?

File: src/test/java/seedu/address/logic/commands/ShortcutCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+    @Test
+    public void executeUndoCommand_invalidCommandWord_throwsCommandException() throws CommandException {
+        thrown.expect(CommandException.class);
+        thrown.expectMessage(ShortcutCommand.MESSAGE_NO_COMMAND_TO_MAP);
+
+        CommandResult commandResult = getAddShortcutForCommand(validTestingShortcutWord,
+                invalidTestingCommandWord,
+                                                               model).executeUndoableCommand();
+    }
```
Comment (randypx) [inline src/test/java/seedu/address/logic/commands/ShortcutCommandTest.java]: large space between line.

File: src/test/java/seedu/address/logic/commands/DeleteShortcutCommandTest.java (hunk 3 above)
Comment (shanmu9898) [reply src/test/java/seedu/address/logic/commands/DeleteShortcutCommandTest.java]: Will correct it in the next PR as the naming has to be changed in multiple places.

**Pull request merged**